index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="index.css">
+    </head>
+    <body>
+        <header class="wrapper">
```

Comment:
all content inside `header` tag is not correct. Create a simple div with class wrapper, inside which you will have all content (header, aside, main, footer)